add_users.sh

```diff
@@ -9,17 +9,22 @@ if [ $(id -u) -eq 0 ]; then
 	do
 		dir=${dir%*/}
 		username=${dir##*/}
+		bash /xal/user_services/enable_ssh.sh $username
```

Comment:
Maybe I misread something, but shouldn't this line run after user is created?

Reply:
I do it this way because it automatically updates a users key if a new one is merged to the internal repo.

Since `enable_ssh` copies the public key to `/home/$USER/.ssh/authorized_keys` calling it before user creation acts as an update in case the public key changed on master after a pull.